cmd/client/main.go

```diff
@@ -170,25 +177,30 @@ func (o *GrpcProxyClientOptions) Validate() error {
 	if o.testDelaySec < 0 {
 		return fmt.Errorf("please do not ask for less than a 0 second delay(%d)", o.testDelaySec)
 	}
+	if o.afterDelaySec < 0 {
+		return fmt.Errorf("please do not ask for less than a 0 second delay(%d)", o.afterDelaySec)
+	}
 	return nil
 }
 
 func newGrpcProxyClientOptions() *GrpcProxyClientOptions {
 	o := GrpcProxyClientOptions{
-		clientCert:   "",
-		clientKey:    "",
-		caCert:       "",
-		requestProto: "http",
-		requestPath:  "success",
-		requestHost:  "localhost",
-		requestPort:  8000,
-		proxyHost:    "localhost",
-		proxyPort:    8090,
-		proxyUdsName: "",
-		mode:         "grpc",
-		userAgent:    "test-client",
-		testRequests: 1,
-		testDelaySec: 0,
+		clientCert:    "",
+		clientKey:     "",
+		caCert:        "",
+		requestProto:  "http",
+		requestPath:   "success",
+		requestHost:   "localhost",
+		requestPort:   8000,
+		proxyHost:     "localhost",
+		proxyPort:     8090,
+		proxyUdsName:  "",
+		mode:          "grpc",
+		userAgent:     "test-client",
+		testRequests:  1,
+		testDelaySec:  0,
+		afterDelaySec: 0,
+		closeIdleConn: true,
```

Comment:
nit: Seems like it would make more sense for the default to be false since that's the default behavior in k/k and in general?

Reply:
Torn. K/K I believe should be calling closeIdleConnection, but I agree it isn't. In case people are using this as an example, I would rather the default was the correct behavior. If the client doesn't close idle connections, then we have to maintain the tunnel all the way through, along with goroutines for each of the segments. The flag is here to allow that to be easily demonstrated.